.github/workflows/coverage_test.yaml

```diff
@@ -16,7 +19,21 @@ jobs:
             - name: Get submodules
               run: git submodule update --init --force --recursive
             - name: Install packages
-              run: sudo apt-get update && sudo apt-get install build-essential libhdf5-dev lcov
+              run: sudo apt-get update && sudo apt-get install build-essential libhdf5-dev
+            - name: Install lcov
+              run: |
+                VERSION="1.15"
```

Comment:
can you add a comment why we can't use the default one?

Reply:
sure